src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.tizen.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace CommunityToolkit.Maui.ApplicationModel;
+
+using Tizen.Applications;
+using Tizen.NUI;
```

Comment:
Let's put the `namespace` below the `using` statements to remain consistent with existing code formatting

```suggestion
using Tizen.Applications;
using Tizen.NUI;

namespace CommunityToolkit.Maui.ApplicationModel;
```